bin/setup.js

```diff
@@ -9,7 +9,19 @@ var isInstallIntoApp = /node_modules/.test(process.cwd())
 // This block only executes if Hoodie is installed with as a dependency
 if (isInstallIntoApp) {
   var pathToAppRoot = path.resolve('..', '..')
-  var packageJson = require(path.join(pathToAppRoot, 'package.json'))
+
+  // log warning if package.json cannot be found (hoodiehq/hoodie#751)
+  try {
+    var packageJson = require(path.join(pathToAppRoot, 'package.json'))
+  } catch (error) {
+    if (error.code !== 'MODULE_NOT_FOUND') {
+      throw error
+    }
+    log.warn('setup', 'Could not find package.json at ' + path.join(pathToAppRoot, 'package.json'))
+    log.warn('setup', 'You must manually set the start script in your app’s package.json to "hoodie" in order for "npm start" to work')
+
+    process.exit(0)
```

Comment:
I didn’t know that `console.log`s or `process.(stderr|stdout).write`s could potentially not finish if we call process.exit(), I think I do that quite a bit. I’d love to learn what the proper way to handle this is. I’m not sure if a `beforeexit` event handler will help because the documentation says 

> The 'beforeExit' event is not emitted for conditions causing explicit termination, such as calling process.exit()